battDB/templates/experiment.html

```diff
@@ -118,7 +118,11 @@ <h5>Measurement data: {{data_file.name}}</h5>
         </tr>
         <tr>
           <td><b>Parsed columns:</b></td>
-          <td>{{ data_file.ts_headers }} </td>
+          <td>{{ data_file.attributes.parsed_columns }} </td>
+        </tr>
+        <tr>
+          <td><b>Original file columns:</b></td>
+          <td>{{ data_file.attributes.parsed_header_columns }} </td>
```

Comment:
Another thing I just noticed. I just tried changing the column name in the file to an invalid one and these two lists were still the same and it didn't include the invalid column name in the "Original file columns" list. Which is not what I expect here from the name.

Reply:
Do you mean that you changed a column header in a raw data file to one that is not listed in the `Parser` and for which there is no mapping? And still the same as what? As each other when it was a valid header? 

Can you provide a bit more info on the input/output you used?

Reply:
Sorry, the "two lists" are the "Original file columns" and the "Parsed Columns". I used the `biologic_example.csv` file and changed the time header from `time/s` to `Time/s` and it failed to parse the file properly, but those two lists were the same, I would've expected to see the incorrect time header in the "Original..." list.

![Screenshot 2023-01-23 at 12 29 05](https://user-images.githubusercontent.com/40875798/214039855-3c571587-de23-429b-aae3-45f1f1f30004.png)

Reply:
Ah I see what you mean. Yes that latter list should probably be labeled "Parsed columns (original file headers)" for clarity. If it's not parsed, it won't get added to either list.

Reply:
I've re-labeled that list and added another one to the template for clarity

![image](https://user-images.githubusercontent.com/5871253/214062918-8fc491dd-bc75-4501-9562-90fcd1d9337a.png)